### PR TITLE
added validation function

### PR DESCRIPTION
**Pull request by asish0512**
**Pull Request Requirements:**

* Provide a brief description of the changeset.
* Include a screenshots if applicable
* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).
* Solves #5176

https://github.com/hcengineering/platform/assets/14013271/281b26da-53a7-4d01-8db9-5daa2db1613c

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjE0ZmVhZTI0N2IzZDJhZDI5MjI2NDYiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.b9S9bgN37kSDDwws7tyDc75BV3Er5GgketObWdcco80">Huly&reg;: <b>UBERF-6440</b></a></sub>

### CODE REVIEW

File: plugins/tags-resources/src/components/CreateTagElement.svelte [hunk 1]
```diff
@@ -94,13 +94,20 @@
       }
     )
   }
+  
+
+  function isNotEmpty(str: string): boolean {
```
Comment (aonnikov) [inline plugins/tags-resources/src/components/CreateTagElement.svelte]: It is idiomatic for use to write something like
```
$: canSave = title.trim().length > 0
```
I'm also not sure regarding `undefined` check, the `str` and `title` types do not assume `undefined` value, so either remove it or fix types.

**Review by aonnikov — CHANGES_REQUESTED** [review]

File: plugins/tags-resources/src/components/CreateTagElement.svelte (hunk 1 above)
Comment (asish0512) [reply plugins/tags-resources/src/components/CreateTagElement.svelte]: @aonnikov 
Thank you for the clarification. Given that the variable title is explicitly declared as a string, an undefined check may indeed be unnecessary for string validation in this specific context. Modifying the PR accordingly.

**Comment by aonnikov** [comment]
Could you please sign-off your commits?

File: plugins/tags-resources/src/components/CreateTagElement.svelte (hunk 1 above)
Comment (asish0512) [reply plugins/tags-resources/src/components/CreateTagElement.svelte]: @aonnikov I have signed off all commits.

**Review by aonnikov — APPROVED** [review]

**Pull request merged**